_posts/en/newsletters/2022-10-05-newsletter.md

```diff
@@ -0,0 +1,203 @@
+---
+title: 'Bitcoin Optech Newsletter #220'
+permalink: /en/newsletters/2022/10/05/
+name: 2022-10-05-newsletter
+slug: 2022-10-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal for new opt-in transaction relay
+rules and summarizes research into helping LN channels stay balanced.
+Also included are our regular sections listing new software releases and
+release candidates plus notable changes to popular Bitcoin infrastructure
+projects.
+
+## News
+
+- **Proposed new transaction relay policies designed for LN-penalty:**
+  Gloria Zhao [posted][zhao tx3] to the Bitcoin-Dev mailing list a proposal to
+  allow transactions to opt-in to a modified set of transaction relay
+  policies.  Any transaction which sets its version parameter to `3`
+  will:
+
+    - Be replaceable while it is unconfirmed by a transaction paying a
+      higher feerate and a higher total fee (the current main
+      [RBF][topic rbf] rules)
+
+    - Be rejected if any of the transactions it depends on (its ancestor
+      transactions) are both unconfirmed and also v3
```

Comment:
I'm confused by the image in an earlier comment conjuction with the current text. It seems that all of the “valid” examples in the image would be in conflict with this rule. Is it possible that there was a mixup here?

Reply:
Thanks @Xekyo! Looks like fixed in 7166a56